docs/settingUp.md

```diff
@@ -111,6 +113,23 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
```

Comment:
add : 

```
./gradlew resetEclipseDeps
```

I think dependencies will change? and also the steps for setting up the code `Formats` or `Indentation .  

If it remains same then i think you need to move these(IntelliJ setup part) above. As it might imply that those steps are only for eclipse.